actions/campaigns.go

```diff
@@ -222,11 +210,22 @@ func GetCampaign(c *gin.Context) {
 	})
 }
 
+type paramsCampaign struct {
```

Comment:
Rename paramsCampaign to postCampaign, to keep consistency.

Reply:
@FilipNikolovski I'm using params coz it's same structure for put and post camp should i make 2 same structures?

Reply:
One structure is okay in this case, we can rename it to `campaignParams` to be consistent.